test-common/meson.build

```diff
@@ -19,11 +19,21 @@ libeos_updater_test_common_headers = [
   'utils.h',
 ]
 
+# ostree-trivial-httpd is in libexecdir/libostree, but we don't know
+# where libexecdir is. Try to build a path for typical places.
+ostree_trivial_httpd = find_program(
+    'ostree-trivial-httpd',
+    join_paths(libexecdir, 'libostree/ostree-trivial-httpd'),
```

Comment:
Unless I’m misinterpreting [your comment](https://github.com/endlessm/eos-updater/pull/281#issuecomment-805113627), I think this needs an additional `join_paths(libexecdir, 'installed-tests/libostree/ostree-trivial-httpd'),` entry then.

Reply:
Sorry, I was a bit terse. I'll update with that location so that it works if/when upstream moves it there.

Reply:
Seems easier to me to add the entry now, then we never have to think about it again, but LGTM either way. Merge whenever you like :)

Reply:
Poor communication on my part. It was my intention to update the PR to reflect the new potential future path. I just hadn't gotten around to it. Updated now.